build.yaml

```diff
@@ -565,10 +651,19 @@ common_cli:
 
   build_vagrant_tutorial_basebox: &build_vagrant_tutorial_basebox |
     vagrant destroy -f
-    parse_logs \$venv/bin/python admin/build-vagrant-box --box tutorial \
-      --branch \${TRIGGERED_BRANCH} --build-server  http://\${eth0_ip}
-    parse_logs vagrant box add --force "clusterhq/flocker-tutorial" \
-      vagrant/tutorial/flocker-tutorial-\${FLOCKER_VERSION}.box
+    (
+      retry_n_times_with_timeout 3 1800 \
+        \$venv/bin/python admin/build-vagrant-box \
+          --box tutorial \
+          --branch \${TRIGGERED_BRANCH} \
+          --build-server http://\${eth0_ip}
+    )|| abort_build
+
+    (
+      retry_n_times_with_timeout 2 1200 \
+        vagrant box add --force "clusterhq/flocker-tutorial" \
+          vagrant/tutorial/flocker-tutorial-\${FLOCKER_VERSION}.box
+    ) || abort_build
```

Comment:
they are on the fat side, however these jobs take a long time to run and only run overnight.
They're not part of the multijob